system/LINK/etc/default/evm

```diff
@@ -25,3 +25,4 @@ export QPID_SSL_CERT_DB=/etc/qpid/certs
 [[ -s /etc/default/evm_bundler ]] && source /etc/default/evm_bundler
 [[ -s /opt/rh/postgresql92/enable ]] && source /opt/rh/postgresql92/enable
 [[ -s /opt/rh/nodejs010/enable ]] && source /opt/rh/nodejs010/enable
+[[ -s /opt/rh/cfme-gemset/enable ]] && source /opt/rh/cfme-gemset/enable
```

Comment:
@jrafanie and @blomquisg I reviewed this with @Fryguy. The name of the script will be /etc/default/evm_productization. Updated PR to come soon.